upcoming_changes/2960.maintenance.rst

```diff
@@ -0,0 +1 @@
+Fix packaging test data and regression introduced in element refactor
```

Comment:
Since #2960 won't be shipped in a release independently from this fix, maybe there is no need to add this in the changelog?

Reply:
Done